Sources/OAuth2Client/NXOAuth2Request.m

```diff
@@ -116,6 +117,24 @@ - (void)performRequestWithSendingProgressHandler:(NXOAuth2ConnectionSendingProgr
     self.me = self;
```

Comment:
This creates a retain cycle. The caller should be responsible for keeping a reference.